vuln/core/README.txt

```diff
@@ -0,0 +1,631 @@
+Node.js v7.1.0. The reverted code was found to contain a potential buffer
+overflow in output written to the console. We are not aware of any exploit of
+this flaw and it only impacts Windows 10 (November update and later). This flaw
+has been assigned the identifier CVE-2016-9551 and was originally discovered and
+reported by Hitesh Kanwathirtha of Microsoft.
+
+Users of the v7 release line running on Windows 10 should upgrade to Node.js
+v7.2.0 at their earliest convenience.
+
+No other version of Node.js is known to be impacted by this flaw.
+
+--
+
+XXX break into 4 records?
+
+ref: http://seclists.org/oss-sec/2016/q4/602
+ref: https://github.com/madler/zlib/commit/6a043145ca6e9c55184013841a67b2fef87e44c0
+cve: CVE-2016-9840.
+ref: https://github.com/madler/zlib/commit/9aaec95e82117c1cb0f9624264c3618fc380cecb
+cve: CVE-2016-9841.
+ref: https://github.com/madler/zlib/commit/e54e1299404101a5a9d0cf5e45512b543967f958
+cve: CVE-2016-9842.
+ref: https://github.com/madler/zlib/commit/d1d577490c15a0c6862473d7576352a9f18ef811
+cve: CVE-2016-9843.
+description: undefined language constructs that may have security impact
+vulnerable: 4.x, 5.x, 6.x, 7.x
+patched: ^4.8.2, ^6.10.2, ^7.6.0
+overview:
+An upgrade to zlib 1.2.11 to fix a number of low severity CVEs
+that were present in zlib 1.2.8.
+
+--
+
+vulnerable: 0.10.x, 0.12.x, 4.x
+patched: ^0.10.48, ^0.12.17, ^4.6.1
+cve: CVE-2016-5180
+description: ares_create_query single byte out of buffer write
+ref: https://nodejs.org/en/blog/vulnerability/october-2016-security-releases/
+ref: https://github.com/nodejs/node/pull/8849
+ref: https://c-ares.haxx.se/adv_20160929.html
+overview:
+A security vulnerability has been discovered in the c-ares library that is
+bundled with all versions of Node.js. Due to the difficulty of triggering and
+making use of this vulnerability we currently consider this a low-severity
+security flaw for Node.js users.
+
+More information at https://c-ares.haxx.se/adv_20160929.html
+
+--
+
+vulnerable: 6.x
+patched: ^6.9.0
+ref: https://nodejs.org/en/blog/vulnerability/october-2016-security-releases/ for
+description: automatically loading OPENSSL_CONF is unsafe
+overview:
+Always triggering a configuration file load attempt from `OPENSSL_CONF` or the
+default location for the current platform may allow an attacker to load
+compromised OpenSSL configuration into a Node.js process if they are able to
+place a file in a default location.
+
+--
+
+vulnerable: 6.x
+patched: ^6.9.0
+ref: https://nodejs.org/en/blog/vulnerability/october-2016-security-releases/ for
+cve: CVE-2016-5172
+overview:
+The V8 parser mishandled scopes, potentially allowing an attacker to obtain
+sensitive information from arbitrary memory locations via crafted JavaScript
+code. This vulnerability would require an attacker to be able to execute
+arbitrary JavaScript code in a Node.js process.
+
+XXX This is not remote, why was it considered a security issue for node?
```

Comment:
I will find the PR for this in a second pass and look for commentary by @nodejs/security